### PR TITLE
Localization (Chinese)

### PR DESCRIPTION
**Pull request by septs**

### CODE REVIEW

**Comment by septs** [comment]
CC @Jack-Works

File: _data/cn/site.yml [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
-  proposals: 语言特点 提议
-  archives: 档案
+  standards: 标准文件
+  proposals: 语言提案
+  archives: 历史存档
   process: 我们的流程
   coc: Code of Conduct
   feedback: 我们正在寻找反馈
-  feedback-prompt:  请<a href="https://github.com/tc39/tc39.github.io/issues/new" target="_blank">在<a href="https://github.com/tc39/tc39.github.io" target="_blank">我们的网站仓库</a>中打开一个问题</a>。
+  feedback-prompt:  请<a href="https://github.com/tc39/tc39.github.io/issues/new" target="_blank">在<a href="https://github.com/tc39/tc39.github.io" target="_blank">我们的网站仓库</a>中创建一个问题</a>。
```
Comment (himself65) [inline _data/cn/site.yml]: the wrong link comparing with the original text

**Comment by codehag** [comment]
thank you so much!

**Comment by septs** [comment]
@codehag ready, thanks

**Pull request merged**